stable/weave-cloud/README.md

```diff
@@ -0,0 +1,40 @@
+# Weave Cloud Agents
+
+Weave Cloud is a add-on to Kubernetes which provides Continuous Delivery, along with hosted Prometheus Monitoring and a visual dashboard for exploring & debugging microservices.
+
+This package contains the agents which connect your cluster to Weave Cloud.
```

Comment:
Can you please add a little more details about what resources are exactly installed on the cluster?